src/main/java/ToDos.java

```diff
@@ -0,0 +1,19 @@
+import java.util.Arrays;
+
+public class ToDos extends Task {
+
+    public ToDos(String d) {
+        super(d);
+        this.type = "T";
+    }
+
+    public ToDos(String d, String done) {
+        super(d);
+        this.type = "T";
```

Comment:
Any reason why not set type to T outside the constructor and not in the constructor?